ci/build.yml

```
@@ -251,6 +251,31 @@ jobs:
         trigger_sha: '$(trigger_sha)'
     - template: report-end.yml
 
+- job: build_canton_3x_with_bazel
+  dependsOn:
+    - check_for_release
+    - build_canton
+  variables:
+    - name: release_sha
+      value: $[ dependencies.check_for_release.outputs['out.release_sha'] ]
+    - template: job-variables.yml
+  timeoutInMinutes: 60
+  pool:
+    name: 'ubuntu_20_04'
+    demands: assignment -equals default
+  steps:
+    - template: report-start.yml
+    - checkout: self
+    - bash: |
+        set -euo pipefail
+        git checkout $(release_sha)
+        ci/build-canton-3x.sh
```

```suggestion
        if [ -x ci/build-canton-3x.sh ]; then
          ci/build-canton-3x.sh
        fi
```
So we can run this on older releases too. At least 2.3 and 2.7 are at risk of getting further patches still.